### DISCUSSION TITLE
not working

### DISCUSSION THREAD

**Discussion by boobythebuilder** [3kh0/ext-remover#35] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
i was trying it but it didn't work i don't know what i did wrong

**Comment by 3kh0** [comment]
can you send screenshots? what method are you using?

**Reply by Aka-but-nice** [reply]
Refer to #348 or #347

**Comment by rusprice** [comment, marked as answer]
talk about vague

**Reply by 3kh0** [reply]
i want to mark that as the answer so badly
